apps/perf-test/tasks/fluentPerfRegressions.ts

```diff
@@ -37,7 +37,7 @@ function linkToFlamegraph(value: string, filename: string) {
 function fluentFabricComparison(perfCounts: any, reporter: Reporter) {
   const results = _.mapValues(
     _.pickBy(perfCounts, (value, key) => key.endsWith('.Fluent')),
-    stats => {
+    (stats) => {
```

Comment:
A lot of this looks like a bad merge where you accidentally ended up running a newer version of prettier on all files in the merge. Can you try reverting the stuff that's not directly related? This should be fixed moving forward by Martin's recent changes to install the proper version of prettier at the repo root.